Cargo.toml

```diff
@@ -3,6 +3,7 @@ members = [
     "api_identity",
     "bootstore",
     "bootstrap-agent-client",
+    "certificates",
```

Comment:
Shouldn't this go into default-members too?

Reply:
Hah, yes, thanks. I think I stopped searching for "passwords" and started searching for "omicron-passwords", which led me to miss default-members. Fixed in 3e16589